tokio-quiche/src/quic/io/worker.rs

```diff
@@ -77,31 +79,48 @@ const RELEASE_TIMER_THRESHOLD: Duration = Duration::from_micros(250);
 const GSO_THRESHOLD: usize = 1_000;
 
 pub struct WriterConfig {
+    /// Connection ID used during the handshake
     pub pending_cid: Option<ConnectionId<'static>>,
+    /// Peer address used during the handshake
     pub peer_addr: SocketAddr,
+    /// Whether to use GSO
     pub with_gso: bool,
+    /// Whether to use GSO with pacing
     pub pacing_offload: bool,
+    /// Whether there are packet info
     pub with_pktinfo: bool,
 }
 
 #[derive(Default)]
 pub(crate) struct WriteState {
+    /// Whether the connection has completed handshaking
     conn_established: bool,
+    /// Number of bytes written for this path
     bytes_written: usize,
+    /// Size of the segment written to the socket
     segment_size: usize,
+    /// Number of packets prepared for this path
     num_pkts: usize,
+    /// Scheduled transmit time for pacings
     tx_time: Option<Instant>,
+    /// Whether there are pending data to send on any path
     has_pending_data: bool,
     // If pacer schedules packets too far into the future, we want to pause
     // sending, until the future arrives
     next_release_time: Option<Instant>,
     // If set, outgoing packets will be sent to the peer from the `send_from`
     // address rather than the listening socket.
+    /// Socket address used as the source for outgoing packets
     send_from: Option<SocketAddr>,
+    /// Socket address used as the destination for outgoing packets
+    send_to: Option<SocketAddr>,
+    /// Path identifier
+    network_path_id: Option<usize>,
```

Comment:
This is not a `network_path_id`, this is a `socket_id`.
```suggestion
    /// Socket identifier
    socket_id: Option<usize>,
```